compiler/luci/pass/src/ResolveCustomOpAddPass.cpp

```diff
@@ -25,30 +25,49 @@
 namespace
 {
 
+/// @brief Returns the number of Custom("BroadcastTo") node among node's inputs
```

Comment:
`node among node's inputs`

> but the word node already exist that may confuse understanding.

Reply:
@seanshpark 

Oh, I'm sorry I missed the comment in the back. 😅

I uploaded it after modifying it as follows, is it okay?

> /// @brief Returns the number of Custom("BroadcastTo") among node's inputs